README.Rmd

```diff
@@ -15,9 +15,8 @@ knitr::opts_chunk$set(
 [![rstudio mirror downloads](http://cranlogs.r-pkg.org/badges/originr)](https://github.com/metacran/cranlogs.app)
 [![cran version](http://www.r-pkg.org/badges/version/originr)](https://cran.r-project.org/package=originr)
 
-Species Origin Data
-
-Data sources:
+## Description 
+This package allows you to obtain species origin data (whether species is native/invasive) from many sources on the web including:
```

Comment:
this could use a little more information please. 

also, perhaps say instead "(whether a species is native, non-native or invasive in a particular region/country)"